### DISCUSSION TITLE
Cloudinary -> Gatsby VS Netlify CMS

### DISCUSSION THREAD

**Discussion by emanuele-em** [gatsbyjs/gatsby#36170] [Help]
I'm struggling with this question:

I'm creating a sort of blog with gatsby and netlify CMS but I came to a crossroads:

Is better **solution (1)** with

- **Cloudinary**
- **Gatbsy** with _gatsby-source-cloudinary_ + _gatsby-transformer-cloudinary_ and _GatsbyImage_
- **Netlify CMS** with standard image uploader

or is better **solution (2)** with

- **Cloudinary**
- **Gatsby** image sourced with the source of the image taken from an external Cloudinary URL
- **Netlify CMS** with Cloudinary _image uploader_

thank you very much

**Comment by jongwooo** [comment]
@emanuele-em I think **solution (1)** is better.
